gradle/java-module.gradle

```diff
@@ -228,17 +228,19 @@ tasks.withType( Test.class ).each { test ->
 
 	test.usesService( project.gradle.sharedServices.registrations.getByName( 'databaseService' ).service )
 
-	// Byteman needs this property to be set, https://developer.jboss.org/thread/274997
-	test.jvmArgs += ["-Djdk.attach.allowAttachSelf=true"]
-
 	test.jvmArgumentProviders.add(
 			new HeapDumpPathProvider( path: project.layout.buildDirectory.dir("OOM-dump") )
 	)
 
-	test.jvmArgs += [
-			'-XX:+HeapDumpOnOutOfMemoryError',
-			'-XX:MetaspaceSize=256M'
-	]
+	// Byteman needs this property to be set, https://developer.jboss.org/thread/274997
+	test.jvmArgs('-Djdk.attach.allowAttachSelf=true')
+
+	if (project.hasProperty('additionalTestJvmArg')) {
+		test.jvmArgs(project.property('additionalTestJvmArg'))
+	}
```

Comment:
I think this is already supported with `-Ptest.jdk.launcher.args=...`.

See https://github.com/hibernate/hibernate-orm/blob/e407c53c861a0e9dc745a78b15a5b1afd9ffd7c7/Jenkinsfile#L127C13-L127C35

Reply:
ohh ok, I didn't see anything in gradle file and wasn't sure, but it seems this is handled here:

https://github.com/hibernate/hibernate-orm/blob/e407c53c861a0e9dc745a78b15a5b1afd9ffd7c7/local-build-plugins/src/main/java/org/hibernate/orm/toolchains/JavaModulePlugin.java#L128-L134

will make an update ...
Thanks 😃

Reply:
> ohh ok, I didn't see anything in gradle file

We try to move as much as possible to (local) plugins to keep the scripts as simple as possible.